File: cs/temp-launcher/app_pojavlauncher/src/main/java/net/kdt/pojavlaunch/yggdrasil/SkinModelType.java
package net.kdt.pojavlaunch.yggdrasil;

public enum SkinModelType {
    NONE(-1),
    STEVE(0),
    ALEX(1);

    private final int targetParity;

    SkinModelType(int targetParity) {
        this.targetParity = targetParity;
    }

    public int getTargetParity() {
        return targetParity;
    }
}

File: cs/app_pojavlauncher/src/main/java/net/kdt/pojavlaunch/skin/SkinModelType.kt
package net.kdt.pojavlaunch.skin

/**
 * Arm model of a Minecraft player skin.
 *
 * [targetParity] is used by LocalUuidUtils to encode the model into the UUID's
 * parity bits so Minecraft can determine the arm width without a session server.
 *
 * STEVE = 4px wide arms (default, classic)
 * ALEX  = 3px wide arms (slim)
 * NONE  = no skin assigned; UUID is returned unmodified
 */
enum class SkinModelType(val targetParity: Int) {
    NONE(-1),
    STEVE(0),
    ALEX(1)
}
